Core/src/org/sleuthkit/autopsy/datamodel/DeletedContent.java

```diff
@@ -427,8 +428,14 @@ static private String makeQuery(DeletedContent.DeletedContentFilter filter) {
                         logger.log(Level.SEVERE, "Unsupported filter type to get deleted content: {0}", filter); //NON-NLS
 
                 }
+                
+                if(UserPreferences.hideKnownFilesInViewsTree()) {
+                    query += " AND (known != " + TskData.FileKnown.KNOWN.getFileKnownValue() //NON-NLS
+                            + " OR known IS NULL)"; //NON-NLS
+                }
```

Comment:
That's correct. It has a limited scope of types it looks at, and SLACK is not part of that scope.